src/components/About.js

```diff
@@ -1,18 +1,38 @@
-import React from "react";
+import React, { useRef, useState, useEffect } from "react";
 import { useHistory } from "react-router-dom";
 
 import textStyles from "./TextSection/text-section.module.css";
```

Comment:
Remember what I said about files location?

- Try to keep CSS close to the JS.
- This component is "About" but it's used in main page only. a better place would be `pages/Main/components/*` for example.

Reply:
You mean the text-section module? It's a module with text styles I imported in most components. It did allow me to reuse most text classes but as the file was getting bigger I did start wondering if it was a good idea. Should I have strictly kept each component's css separate (and duplicate styles), moved reused styles to App.css, or something else?

Reply:
I see. You need to create a typography theme, as you did with colors. [see example](https://medium.com/codyhouse/create-your-design-system-part-1-typography-7c630d9092bd). So the most common typography styles would go to `typography.module.scss`. Stuff more specific/exceptions, would go to their own files.